package.json

```diff
@@ -7,6 +7,7 @@
     "npm": ">=7.24.2"
   },
   "dependencies": {
+    "alephium-js": "1.0.5",
```

Comment:
@mvaivre, since I had to update the API schema for this feature, I took the opportunity to integrate alephium-js slowly (one use-case at the time).

Reply:
Good stuff!